cookbooks/aws-parallelcluster-slurm/files/default/config_slurm/scripts/health_checks/gpu_health_check.sh

```diff
@@ -44,12 +44,14 @@ function main() {
   command -v dcgmi  &>/dev/null
   fast_success_exit "$?" "The Gpu Health Check has been executed but the NVIDIA DCGM Diagnostic tool is not available in this system or is located in a different file path. Please consider using an official ParallelCluster AMI."
 
-  ## Check if there are GPU associated to the job
+  ## Check if the NVIDIA GPU is supported by DCGM tool
+  nvidia_smi_output=$(nvidia-smi -L)
```

Comment:
Can we log this output (maybe we already log this somewhere else in the script)? It may be useful for debugging (especially if the `is_dcgm_supported` function returns `1`)

Reply:
done, done will be
```
2023-04-27 12:24:58,891 - [gpu_health_check.sh] - INFO - JobID 82 - GPU list is 'GPU 0: Tesla T4 (UUID: GPU-bb17db30-337e-f752-b3c9-d4525868d411), GPU 1: Tesla T4 (UUID: GPU-2a09030e-a9ab-4b49-eb95-5bff08da9472), GPU 2: Tesla T4 (UUID: GPU-61662417-a573-04a4-1dce-4153b45efa83), GPU 3: Tesla T4 (UUID: GPU-e994535f-8505-e40b-4cc6-063d1d36535b)'
```